elasticsearch/install.sh

```diff
@@ -8,10 +8,20 @@ ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
 /usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-ssl/${SG_SSL_VER}
 /usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-2/${SG_VER}
 /usr/share/elasticsearch/bin/plugin install io.fabric8/elasticsearch-cloud-kubernetes/${ES_CLOUD_K8S_VER}
-/usr/share/elasticsearch/bin/plugin install io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+#/usr/share/elasticsearch/bin/plugin install io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+/usr/share/elasticsearch/bin/plugin install file:///openshift-elasticsearch-plugin-2.4.1.5-SNAPSHOT.zip
```

Comment:
This is really temporary until we publish the plugin.  Ill update when that merges and we agree things are good